programs/tigervnc.json

```diff
@@ -2,8 +2,8 @@
     "files": [
         {
             "path": "$HOME/.vnc",
-            "movable": false,
-            "help": "Currently unsupported.\n\n_Relevant issue:_ https://github.com/TigerVNC/tigervnc/issues/1195\n"
+            "movable": true,
+            "help": "Supported by default.\n\nNewer versions of tigervnc automatically create directories at correct locations, consult `man vncviewer` to see where to move files.\n**WARNING: Failure to move files to the correct location can result in locking out VNC access**"
```

Comment:
```suggestion
            "help": "Supported by default.\n\nAll versions above 1.14.0 will automatically create directories at correct locations, consult `man vncviewer` to see where to move files.\n**WARNING: Failure to move files to the correct location can result in locking out VNC access**"
```

Reply:
I don't know whether it should not be changed to 1.13.1, since the change was introduced in [1.14.0](https://github.com/TigerVNC/tigervnc/releases/tag/v1.14.0).